kyo-core/shared/src/main/scala/kyo/KyoApp.scala

```diff
@@ -34,12 +34,17 @@ object KyoApp:
             initCode += (() => handle(v))
     end Base
 
-    type Effects = Fibers & Resources
+    type Effects = Fibers & Resources & Aborts[Throwable]
+
+    def trying[T](timeout: Duration)(v: T < Effects)(
```

Comment:
What do you think of naming this method `attempt`? We already have `IOs.attempt`.

Reply:
Sure, that seems more idiomatic.